platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -133,5 +137,8 @@ public record PlatformSchedulersConfig(
         @ConfigProperty(defaultValue = "1000") int completeStateNotifierUnhandledCapacity,
         @ConfigProperty(defaultValue = "SEQUENTIAL_THREAD") TaskSchedulerType stateHasherSchedulerType,
         @ConfigProperty(defaultValue = "2") int stateHasherUnhandledCapacity,
+        @ConfigProperty(defaultValue = "SEQUENTIAL CAPACITY(60) UNHANDLED_TASK_METRIC")
+                TaskSchedulerConfiguration stateGarbageCollector,
+        @ConfigProperty(defaultValue = "200ms") Duration stateGarbageCollectorHeartbeatPeriod,
```

Comment:
This is probably from some benchmark but I'm curious to know how we arrived at capacity of 60 and 200ms heartbeat interval ... as being ideal?

Reply:
These are numbers I made up because they seemed reasonable. Unless we see signals during testing that these numbers are causing problems, we probably won't change them.

In general, we don't currently have the infrastructure to decide the values for things like this experimentally ahead of time.